openlr/paths_connector.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "openlr/graph.hpp"
+#include "openlr/openlr_model.hpp"
+#include "openlr/stats.hpp"
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+namespace openlr
+{
+class PathsConnector
+{
+public:
+  PathsConnector(double const pathLengthTolerance, Graph const & graph, v2::Stats & stat)
+    : m_pathLengthTolerance(pathLengthTolerance), m_graph(graph), m_stat(stat)
+  {
+  }
+
+  bool ConnectCandidates(std::vector<LocationReferencePoint> const & points,
+                         std::vector<std::vector<Graph::EdgeVector>> const & lineCandidates,
+                         std::vector<Graph::EdgeVector> & resultPath);
+
+private:
+  bool FindShortestPath(FunctionalRoadClass const frc, uint32_t const maxPathLength,
+                        Graph::Edge const & from, Graph::Edge const & to, Graph::EdgeVector & path);
+
+  bool ConnectAdjacentCandidateLines(FunctionalRoadClass const frc,
```

Comment:
Please, move from and to to the beginning.

Reply:
Done.